test/saithriftv2/Makefile

```diff
@@ -0,0 +1,118 @@
+CXX=g++
+ODIR = ./obj
+THRIFT = /usr/bin/thrift
+
+PREDIR := ../
+
+METADIR := $(PREDIR)$(PREDIR)meta/
+
+SAI_PREFIX = /usr
+SAI_HEADER_DIR ?= $(SAI_PREFIX)/include/sai
+SAI_LIBRARY_DIR ?= $(SAI_PREFIX)/lib
+SAI_HEADERS = $(SAI_HEADER_DIR)/sai*.h
+
+LIBS = -lthrift -lpthread
+LDFLAGS = -L$(SAI_LIBRARY_DIR) -Wl,-rpath=$(SAI_LIBRARY_DIR)
+CPPFLAGS = -I$(SAI_HEADER_DIR) -I. -std=c++11
+ifeq ($(DEBUG),1)
+CPPFLAGS += -O0 -ggdb
+endif
+
+ifeq ($(platform),MLNX)
+CDEFS = -DMLNXSAI
+else
+ifeq ($(platform),BFT)
+CDEFS = -DBFTSAI
+else
+ifeq ($(platform),CAVIUM)
+CDEFS = -DCAVIUMSAI
+else
+CDEFS = -DBRCMSAI
+endif
+endif
+endif
+
+ifneq (, $(wildcard /usr/local/bin/ctypesgen))
+CTYPESGEN = /usr/local/bin/ctypesgen
+else
+CTYPESGEN = /usr/local/bin/ctypesgen.py
+endif
+
+ifeq ($(platform),vs)
+LIBS += -lsaivs -lsaimeta -lsaimetadata -lzmq
+else
+LIBS += -lsai
+endif
+
+
+CPP_SOURCES = 	gen-cpp/sai_constants.cpp \
+				gen-cpp/sai_constants.h \
+				gen-cpp/sai_rpc.cpp \
+				gen-cpp/sai_rpc.h \
+				gen-cpp/sai_types.cpp \
+				gen-cpp/sai_types.h
+
+PY_SOURCES = 	gen-py/__init__.py \
+				gen-py/sai/constants.py \
+				gen-py/sai/__init__.py \
+				gen-py/sai/switch_sai_rpc.py \
```

Comment:
This is the old file. New file is gen-py/sai/sai_rpc.py and gen-py/sai/sai_rpc-remote